packages/core/contracts/financial-templates/perpetual-multiparty/PerpetualPositionManager.sol

```diff
@@ -149,13 +151,23 @@ contract PerpetualPositionManager is FeePayer, AdministrateeInterface {
         address _collateralAddress,
         address _finderAddress,
         bytes32 _priceIdentifier,
+        bytes32 _fundingRateIdentifier,
         string memory _syntheticName,
         string memory _syntheticSymbol,
         address _tokenFactoryAddress,
         FixedPoint.Unsigned memory _minSponsorTokens,
         address _timerAddress,
         address _excessTokenBeneficiary
-    ) public FeePayer(_collateralAddress, _finderAddress, _timerAddress) nonReentrant() {
+    )
+        public
+        FeePayer(_collateralAddress, _finderAddress, _timerAddress)
+        FundingRateApplier(
```

Comment:
this is the change where the `FundingRateApplier` gets the latest implementation address of the  `FinancialProductFinder`.